dev-tools/mage/pkgtypes.go

```diff
@@ -690,13 +691,16 @@ func PackageTarGz(spec PackageSpec) error {
 	return nil
 }
 
-func replaceFileArch(filename string, pkgFile PackageFile, arch string) (string, PackageFile) {
-	filename = strings.ReplaceAll(filename, "universal", arch)
-	pkgFile.Source = strings.ReplaceAll(pkgFile.Source, "universal", arch)
-	pkgFile.Target = strings.ReplaceAll(pkgFile.Target, "universal", arch)
-
-	return filename, pkgFile
-}
+// This function call was commented out in the above PackageTarGz() function.
+// Leave it here in case it gets used again in the future.
+//
+//func replaceFileArch(filename string, pkgFile PackageFile, arch string) (string, PackageFile) {
+//	filename = strings.ReplaceAll(filename, "universal", arch)
+//	pkgFile.Source = strings.ReplaceAll(pkgFile.Source, "universal", arch)
+//	pkgFile.Target = strings.ReplaceAll(pkgFile.Target, "universal", arch)
+//
+//	return filename, pkgFile
+//}
```

Comment:
```suggestion
```

I'm on the side of just deleting code instead of commenting it out, just to keep files from becoming a commented out code graveyard.